### PR TITLE
Update 1st classes

### PR DESCRIPTION
**Pull request by haurenburu**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
**1. Swordsman**
*1.1 Magnum Break*
- Reduces delay after skill from 2 seconds to 0.5 seconds.
- Adds 2 seconds cooldown.
 

**2. Merchant**
*2.1 Crazy Uproar*
- Adds 0.5 seconds fixed casting time.
- Adds 1 second variable casting time.
- Changes skill mechanism, grants Str + 4 and Atk + 30 to the user and nearby party members.
- Adds 30 seconds cooldown.

**3. Mage**
*3.1 Fire Bolt/ Cold Bolt/ Lightning Bolt*
- Increases fixed casting time from 0.88 seconds to 1.2 seconds (on level 10).
- Reduces variable casting time from 3.52 seconds to 3.2 seconds (on level 10).
- Greatly reduces delay after skill.

*3.2 Thunderstorm*
- Reduces fixed casting time from 1.6 seconds to 1.5 seconds (on level 10).
- Reduces variable casting time from 6.4 seconds to 4.5 seconds (on level 10).

*3.3 Fire Ball*
- Fixed casting time and variable casting time of level 1-5 is changed to be the same as level 6-10 (0.2/0.8 seconds respectively)
- Reduces delay after skill to 0.7 seconds on every skill level.

*3.4 Soul Strike*
- Delay after skill is 1.4 seconds on every skill level (reduces from 2.5 seconds to 1.4 seconds on level 10).

*3.5 Frost Diver*
- Reduces delay after skill from 1.5 seconds to 0.5 seconds.

**4. Acolyte**
*4.1 Blessing*
- Also increases Hit by 2 per skill level (Also applies on Archbishop's Clementia too).

*4.2 Increases Agility*
- Also increases attack speed (reduces delay after attack by 1%) per skill level (Also applies on Archbishop's Cantocandidus too)

*4.3 Angelus*
- Also increases MaxHP by 50 per skill level.

*4.4 Holy Light*
- Reduces fixed casting time and variable casting time by 50%.

**Issues addressed:** #2725 

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by tlacson7** [comment]
U mean pre-re will also take this? I hope not.

**Comment by Kenpachi2k13** [comment]
> Change some skills behaviors to match kro

Could you please change this to a list of changes contained in this PR? This would simplify the review a lot.

**Comment by haurenburu** [comment]
@Kenpachi2k13  sorry for the delay to edit :D

@tlacson7  no, all changes comes only to renewal

File: src/map/status.c [hunk 1]
```diff
@@ -248,7 +248,7 @@ static void initChangeTables(void)
 	status->set_sc( SM_AUTOBERSERK       , SC_AUTOBERSERK  , SCB_NONE );
 	add_sc( TF_SPRINKLESAND      , SC_BLIND           );
 	add_sc( TF_THROWSTONE        , SC_STUN            );
-	status->set_sc( MC_LOUD              , SC_SHOUT        , SCB_STR );
+	status->set_sc( MC_LOUD              , SC_SHOUT        , SCB_STR ); // i made a mistake here, aparently SCB_WATAK don`t need to be pass
```
Comment (EPuncker) [inline src/map/status.c:251]: this comment is confusing, I didn't get it

File: src/map/status.c [hunk 2]
```diff
@@ -6112,6 +6120,11 @@ static short status_calc_aspd_rate(struct block_list *bl, struct status_change *
 		aspd_rate += sc->data[SC_STEAMPACK]->val2 * 10;
 	if (sc->data[SC_SKF_ASPD] != NULL)
 		aspd_rate -= sc->data[SC_SKF_ASPD]->val1 * 10;
+	//TODO: test the aspd inc
+#ifdef RENEWAL
+	if (sc->data[SC_INC_AGI])
+			aspd_rate += sc->data[SC_INC_AGI]->val1 * 10;
```
Comment (EPuncker) [inline src/map/status.c:6126]: remove a TAB here so it stay with the same indentation as the rest of the code

**Review by EPuncker — CHANGES_REQUESTED** [review]

File: src/map/status.c (hunk 1 above)
Comment (haurenburu) [reply src/map/status.c:251]: i THINK on my first implementation i set SCB_STR|SBC_WATK but realize that isn't needed for the atk bonus

Comment (ADM-Nax) [reply src/map/status.c:251]: After I applied the diff I got an error where the +ATK wouldn't count unless I unequiped and equiped the weapon. Therefore I thinks is better to set the SCB_* here.
I also have changed the SCB_WATK to SCB_BATK and now its working fine. So the new line should be:

status->set_sc( MC_LOUD              , SC_SHOUT        , SCB_STR | SCB_BATK );

File: src/map/status.c [hunk 3]
```diff
@@ -5008,7 +5008,10 @@ static int status_calc_watk(struct block_list *bl, struct status_change *sc, int
 		watk += sc->data[SC_FLASHCOMBO]->val2;
 	if (sc->data[SC_CATNIPPOWDER])
 		watk -= watk * sc->data[SC_CATNIPPOWDER]->val2 / 100;
-
+#ifdef RENEWAL
+	if (sc->data[SC_SHOUT])
+		watk += sc->data[SC_SHOUT]->val2;
```
Comment (ADM-Nax) [inline src/map/status.c:5013]: I have changed this watk to batk for a better use:

```
	if (sc->data[SC_SHOUT])
		batk += sc->data[SC_SHOUT]->val2;	
```
		
But this should be added under the batk section, therefore after (line 5065 in my case - may differ):
```
	if (sc->data[SC_SHRIMP])
		batk += batk * sc->data[SC_SHRIMP]->val2 / 100;	
```

File: src/map/status.c [hunk 4]
```diff
@@ -199,11 +199,11 @@ static void initChangeTables(void)
 	add_sc( MG_STONECURSE        , SC_STONE           );
 	add_sc( AL_RUWACH            , SC_RUWACH          );
 	add_sc( AL_PNEUMA            , SC_PNEUMA          );
-	status->set_sc( AL_INCAGI            , SC_INC_AGI         , SCB_AGI|SCB_SPEED );
+	status->set_sc( AL_INCAGI            , SC_INC_AGI         , SCB_AGI|SCB_SPEED);
```
Comment (ADM-Nax) [inline src/map/status.c:202]: missing SCB_ASPD:
status->set_sc( AL_INCAGI            , SC_INC_AGI         , SCB_AGI|SCB_SPEED|SCB_ASPD );

File: src/map/status.c [hunk 5]
```diff
@@ -6112,6 +6120,11 @@ static short status_calc_aspd_rate(struct block_list *bl, struct status_change *
 		aspd_rate += sc->data[SC_STEAMPACK]->val2 * 10;
 	if (sc->data[SC_SKF_ASPD] != NULL)
 		aspd_rate -= sc->data[SC_SKF_ASPD]->val1 * 10;
+	//TODO: test the aspd inc
+#ifdef RENEWAL
+	if (sc->data[SC_INC_AGI])
+		aspd_rate += sc->data[SC_INC_AGI]->val1 * 10;
+#endif
```
Comment (ADM-Nax) [inline src/map/status.c:6127]: The formula is not working properly.
I Have changed to (under the aspd bonus):

```
		if (sc->data[SC_INCREASEAGI])
			bonus += sc->data[SC_INC_AGI]->val1;
```
in the bottom of the section `**// ASPD percentage values**`

**Comment by MishimaHaruna** [comment]
Superseded by #3218 (currently available in the `rebalance` branch for preliminary testing)

**Pull request closed without merging**